WebApp/Development/CompiledIcons.ashx

```diff
@@ -29,7 +29,7 @@ public class CompiledIcons : IHttpHandler
   
   public void ProcessRequest(HttpContext context)
   {
-    string imagePath = context.Server.MapPath("~/Development/Images/UI/Icons");
+    string imagePath = context.Server.MapPath("~/Development/Images/Icons");
```

Comment:
why this changed from UI.

Reply:
this is for generating all image in single Image file . 
now its change .

Reply:
Ok Seems Good